src/main/kotlin/org/jitsi/nlj/transform/node/incoming/VideoParser.kt

```diff
@@ -71,6 +75,7 @@ class VideoParser(
                 return it
             }
         }
+        logger.warn("Unable to find encoding matching packet! packet=$packet, encodings=\n${tracks.joinToString(separator = "\n")}")
```

Comment:
I realized after we were testing this that this log message is a bit confusing if the `tracks` array is empty (the only sign of this is a blank line after the log message).  Do we want to print the number of encodings, or handle the empty array specially, or something?

Reply:
I've seen it too, but don't find it confusing that it prints nothing when it's empty.

Reply:
Ok.